Doc/library/os.rst

```diff
@@ -249,6 +242,15 @@ process and user.
       Updated to support :pep:`584`'s merge (``|``) and update (``|=``) operators.
 
 
+.. function:: reload_environ()
+
+   Update :data:`os.environ` and :data:`os.environb` with changes to the
+   environment made by :func:`os.putenv`, by :func:`os.unsetenv`, or made
```

Comment:
I think that it is better to say "the process environment".
os.putenv and os.unsetenv update the cache, so there is no need to reload after them. You should refer to the C functions.
Please add a note that this function is not thread safe. Calling it while the environment is modified in other thread has undefined behavior. Reading from os.environ or calling os.getenv during reloading can return empty result.

Reply:
If the process environment is mentioned, perhaps it can be specifically mentioned that it is the current process environment (I think)

Reply:
`os.putenv()` and `os.unsetenv()` don't update `os.environ`: see `test_reload_environ()`.